boxes/boxes/vanilla/src/contracts/src/main.nr

```diff
@@ -20,23 +20,21 @@ contract Vanilla {
     fn constructor(
         number: Field,
         owner: AztecAddress,
-        owner_npk_m_hash: Field,
         owner_ovpk_m: OvpkM
     ) {
         let numbers = storage.numbers;
-        let mut new_number = ValueNote::new(number, owner_npk_m_hash);
+        let mut new_number = ValueNote::new(number, owner);
```

Comment:
Out of scope for this task, but does this mean we could retrieve the owner's ovpk_m from `get_public_keys` so the user wouldn't have to awkwardly provide them externally?